### ISSUE TITLE
[UI] : Implement Mobile Responsiveness for Home page

### ISSUE THREAD

**Issue by Riddhiagrawal001** [juspay/hyperswitch#6127]
### Description 
Home page need to be optimised for mobile devices to ensure a seamless user experience across various screen sizes and resolutions.

### Pages to Update:
1. Home page (Route:/dashboard/home)

### Requirements
1. Ensure all elements (text, images, buttons, etc.) scale appropriately for mobile viewports.
2. Changes for mobile responsiveness doesn't hamper the existing web view 
3. Test across multiple devices and browsers (iOS, Android, Chrome, Safari, etc.).

### Current pages in mobile view 
<img width="414" alt="image" src="https://github.com/user-attachments/assets/bb435ebc-dd8d-4b7d-ae07-e55a21f718bd">

### Acceptance Criteria
1. Each page must be fully functional and visually optimised for mobile devices.
5. Navigation elements (menus, buttons) should be easily clickable and accessible on touchscreens.
6. The design should maintain visual consistency with the desktop version while being mobile-friendly.
7. Proper testing and verification on different devices and screen sizes.
8. Changes for mobile responsiveness doesn't hamper the existing web view 

### Submission Process:
- Ask the maintainers for assignment of the issue, you can request for assignment by commenting on the issue itself.
- Once assigned, submit a pull request (PR).
- Maintainers will review and provide feedback, if any. 
- Maintainers can unassign issues due to inactivity, [read more here](https://github.com/juspay/hyperswitch/wiki/Hacktoberfest-Contribution-Rules).

Refer [here](https://github.com/juspay/hyperswitch/blob/main/docs/TERMS_OF_CONTEST.md) for Terms and conditions for the contest.

**Comment by Vaibhav91one** [comment]
Please assign this issue to me.

**Comment by Riddhiagrawal001** [comment]
Hi @Vaibhav91one , I have assigned the issue to you, and it is now ready for you to pick up . Let me know if you have issues regarding this

**Comment by Vaibhav91one** [comment]
Thanks @Riddhiagrawal001. Will let you know if I have any queires.

**Comment by Vaibhav91one** [comment]
Hi @Riddhiagrawal001, I am having a little trouble with finding the right files. I found the component to work with in the following repo, [control center](https://github.com/juspay/hyperswitch-control-center). Please do guide.

**Comment by gorakhnathy7** [comment]
yes @Vaibhav91one You need to submit a PR on https://github.com/juspay/hyperswitch-control-center repo itself.

**Comment by Vaibhav91one** [comment]
Thanks @gorakhnathy7  for the clarification.

**Comment by Vaibhav91one** [comment]
Hi @gorakhnathy7, when I do npm run start, I get the following error. Is the start script using environment variables by an chance.

**Comment by Vaibhav91one** [comment]
@gorakhnathy7 Attached the ss, I dont know how it got removed. I was just checking for your response and saw that ss is not present. 😂😂

**Comment by Vaibhav91one** [comment]
Hi @gorakhnathy7 , @Riddhiagrawal001 Does this work ? and also should I make changes in `.res` or in `.res.js` 
![fix-1](https://github.com/user-attachments/assets/1b6de052-49b5-4262-8d36-042cc95d83f3)

**Comment by Vaibhav91one** [comment]
Please provide feedback @gorakhnathy7 @Riddhiagrawal001.

**Comment by Riddhiagrawal001** [comment]
> Hi @gorakhnathy7 , @Riddhiagrawal001 Does this work ? and also should I make changes in .res or in .res.js.

Also make changes in `.res` file as `.res.js` is the trasnpiled version of .res
Yes this does looks fine . You can add some padding at the top to look better in mobile version

**Comment by Vaibhav91one** [comment]
> > Hi @gorakhnathy7 , @Riddhiagrawal001 Does this work ? and also should I make changes in .res or in .res.js.
> 
> Also make changes in `.res` file as `.res.js` is the trasnpiled version of .res Yes this does looks fine . You can add some padding at the top to look better in mobile version

Will do @Riddhiagrawal001.

**Comment by Vaibhav91one** [comment]
@Riddhiagrawal001, Raised [PR Request](https://github.com/juspay/hyperswitch-control-center/pull/1645)
